mobius/src/interpolation.rs

```diff
@@ -1,3 +1,9 @@
+use crate::Complex;
+
 pub fn lerp(a: f64, b: f64, t: f64) -> f64 {
```

Comment:
test cases:

- lerp with 0 returns a
- lerp with 1 returns b
- lerp with 0.5 returns midpoint
- lerp with 0.75 returns blend of a and b
- lerp with t > 1 returns point beyond b
- lerp with t < 0 returns point beyond a

Reply:
lerp with t infinity returns infinity
lerp with t neg infinity returns neg infinity